auto_nag/scripts/mismatch-priority-tracking-esr.py

```diff
@@ -43,10 +43,10 @@ def get_bz_params(self, date):
             ],
             'priority': ['P3', 'P4', 'P5'],
             'f1': 'OP',
-            'f2': 'cf_tracking_firefox' + esr_version,
+            'f2': 'cf_tracking_firefox_' + esr_version,
             'o2': 'anyexact',
             'v2': ','.join(['+', 'blocking']),
-            'f3': 'cf_status_firefox' + esr_version,
+            'f3': 'cf_status_firefox_' + esr_version,
```

Comment:
maybe you could use: https://github.com/mozilla/relman-auto-nag/blob/master/auto_nag/utils.py#L110

Reply:
better indeed.